pkg/console/operator/sync_v400.go

```diff
@@ -82,30 +83,30 @@ func (co *consoleOperator) sync_v400(updatedOperatorConfig *operatorv1.Console,
 	customLogoCanMount, customLogoError := co.SyncCustomLogoConfigMap(updatedOperatorConfig)
 	// If the custom logo sync fails for any reason, we are degraded, not progressing.
 	// The sync loop may not settle, we are unable to honor it in current state.
-	co.HandleDegraded(updatedOperatorConfig, "CustomLogoConfigMap", customLogoError)
+	co.HandleDegraded(updatedOperatorConfig, ConsoleCustomLogoConfigmapSync, customLogoError)
```

Comment:
Thinking if we should be calling `HandleProgressingOrDegraded` or `HandleAvailable` ?